### PR TITLE
Task bot 2803 add filter to type disruption

### PR DESCRIPTION
**Pull request by pthegner**
# Description

This PR add the filter "with_disruption_types" and "without_disruption_types" for the API disruptions/_search and impacts/_search

See [swagger](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/CanalTP/Chaos/task-BOT-2803-add-filter-to-type-disruption/documentation/swagger.yml)

## Issue

Issue link: BOT-2803

### CODE REVIEW

File: documentation/swagger.yml [hunk 1]
```diff
@@ -2875,6 +2876,20 @@ components:
           items:
             type: string
             format: uuid
+        with_disruption_types:
+          description: Filter by type (include)
+          type: array
+          items:
+            type: string
+            enum:
+              - expected
```
Comment (Fahmus) [inline documentation/swagger.yml:2885]: unexpected

File: documentation/swagger.yml [hunk 2]
```diff
@@ -2875,6 +2876,20 @@ components:
...
+        without_disruption_types:
+          description: Filter by type (exclude)
+          type: array
+          items:
+            type: string
+            enum:
+              - expected
```
Comment (Fahmus) [inline documentation/swagger.yml:2892]: unexpected

File: documentation/swagger.yml [hunk 3]
```diff
@@ -3243,6 +3258,20 @@ components:
           format: uuid
         ptObjectFilter:
           $ref: "#/components/schemas/pt_object_filter"
+        with_disruption_types:
+          description: Filter by type (include)
+          type: array
+          items:
+            type: string
+            enum:
+              - expected
```
Comment (Fahmus) [inline documentation/swagger.yml:3267]: unexpected

File: documentation/swagger.yml [hunk 4]
```diff
@@ -3243,6 +3258,20 @@ components:
...
+        without_disruption_types:
+          description: Filter by type (exclude)
+          type: array
+          items:
+            type: string
+            enum:
+              - expected
```
Comment (Fahmus) [inline documentation/swagger.yml:3274]: unexpected

**Comment by Fahmus** [comment]
Work for me :+1:

**Review by tchevily — APPROVED** [review]

**Pull request merged**